### PR TITLE
LPS-197512 Get the label using current locale before using site default locale on LabelMap

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://liferay.atlassian.net/browse/LPS-197512

### CODE REVIEW

File: portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowTransitionTest.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+		defaultWorkflowTransition.setLabelMap(new HashMap<>());
+		defaultWorkflowTransition.setName("businessRuleReview");
+
+		Assert.assertEquals(
+			"businessRuleReview",
+			defaultWorkflowTransition.getLabel(LocaleUtil.BRAZIL));
+
+		defaultWorkflowTransition.setLabelMap(
+			HashMapBuilder.put(
+				LocaleUtil.US, "Business Rule Review"
```
Comment (p-albuquerque) [inline portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowTransitionTest.java:71]: Question:
I notice that, in the labelMap you set the key passing `LocaleUtil.US`
In the [Assertions](https://github.com/liferay-workflow/liferay-portal/pull/1039/commits/a4624e158724a060714b923d17c2edffdba76435#diff-cc610839626999a12eec1cb669c5d88281d74ed45454a67a91ba8a46cee5827dR52-R54) and [mockito](https://github.com/liferay-workflow/liferay-portal/pull/1039/commits/a4624e158724a060714b923d17c2edffdba76435#diff-cc610839626999a12eec1cb669c5d88281d74ed45454a67a91ba8a46cee5827dR38-R39)
you use the key passing `LocaleUtil.getSiteDefault()`.
Its okay letting this two ways to refer the locale "en-US"?

**Review by p-albuquerque — CHANGES_REQUESTED** [review]
Just a question

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:sf

File: portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowTransitionTest.java (hunk 1 above)
Comment (marcelabc) [reply portal-kernel/test/unit/com/liferay/portal/kernel/workflow/DefaultWorkflowTransitionTest.java]: Thank you for noticing :smile: 
I changed to use only `LocaleUtil.getSiteDefault()`.

**Review by p-albuquerque — APPROVED** [review]

**Comment by p-albuquerque** [comment]
LGTM!
Thank you @marcelabc for the support with this issue!

**Comment by marcelabc** [comment]
ci:reevaluate:1317401_10167

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
Checking CI results...

**Comment by p-albuquerque** [comment]
Related test failures found.
Working on the needed adjusts...
A new PR will be sent soon

**Comment by p-albuquerque** [comment]
Resent in: https://github.com/liferay-workflow/liferay-portal/pull/1042

**Pull request closed without merging**